docs/sdks_03_safe_apps.md

```diff
@@ -81,4 +81,4 @@ safe@gnosis.io or via our [Discord](https://discord.gg/FPMRAwK)
 
 ### Disclaimer
 
-It is at the sole discretion of Gnosis to list or delist apps from the Safe Multisig. Safe Apps are not owned, controlled, maintained or audited by Gnosis.
+It is at the sole discretion of Gnosis to list or delist apps from the Safe Multisig. Safe Apps are not owned, controlled, maintained, or audited by Gnosis.
```

Comment:
Is the comma here really required?

I googled a little bit and it seems that the comma before `or` is required only if clauses are independent (https://www.businessinsider.com/a-guide-to-proper-comma-use-2013-9)

These seem dependent to me